Libs/DICOM/Core/ctkDICOMDatabase.cpp

```diff
@@ -1038,6 +1038,19 @@ QString ctkDICOMDatabase::groupElementToTag(const unsigned short& group, const u
 // methods related to insert
 //
 
+//------------------------------------------------------------------------------
+void ctkDICOMDatabase::prepareInsert()
```

Comment:
I suggest to rename the function to `resetCache` or similar name. The name `prepareInsert` implies it should be called before every insert.

Reply:
maye be `resetCachedIDs()`

Reply:
ID caching is an internal affair of the database. It would be better if users would not be aware of such internal detail (also because the mechanism may change in the future; we may do more or less things to prepare before insertions are started).

Reply:
Fair enough. Let's keep it then.